app/notifiers/publishing_api_notifier.rb

```diff
@@ -13,7 +13,10 @@ def initialize(tag)
 
   def send_single_tag_to_publishing_api
     if tag.published?
-      publishing_api.put_content_item(presenter.base_path, presenter.render_for_publishing_api)
+      unless tag.archived?
```

Comment:
This logic would probably be clearer as a case statement now that there are more than 2 possible states.